R/opts.R

```diff
@@ -70,14 +75,29 @@ offspring_opts <- function(community, isolated, asymptomatic = community) {
 #'   incubation_period = \(n) rweibull(n = n, shape = 2.32, scale = 6.49),
 #'   onset_to_isolation = \(n) rweibull(n = n, shape = 1.65, scale = 4.28)
 #' )
-delay_opts <- function(incubation_period, onset_to_isolation) {
+delay_opts <- function(incubation_period,
+                       onset_to_isolation,
+                       latent_period = 0) {
 
   check_dist_func(incubation_period, dist_name = "incubation_period")
   check_dist_func(onset_to_isolation, dist_name = "onset_to_isolation")
+  checkmate::assert_number(latent_period, lower = 0, finite = TRUE)
+
+  if (latent_period > 0) {
+    warning(
+      "A latent_period > 0 may cause the proportion of presymptomatic ",
```

Comment:
```suggestion
      "A latent_period > 0 causes the proportion of presymptomatic ",
```

this is always the case unless the PDF of the incubation period is exactly zero below `latent_period`, right?

Reply:
OK - as an aside this would be a great case for a test I think.